### ISSUE TITLE
About the accuracy during the pre-training stage

### ISSUE THREAD

**Issue by pprp** [enyac-group/supmae#4]
Hi,  thank you for your great work！

I can't find the results of the classification branch in your paper. 

Can you share the final accuracy of the classification branch during the pre-training stage?

**Comment by Jeff-LiangF** [comment]
@pprp Do you mean the ImageNet validation accuracy during pre-training? I didn't test it yet. But I think we can get some performance but may not be that good . I'll update that result in the following days. Stay tuned!

**Comment by pprp** [comment]
@Jeff-LiangF Thank you for your reply! In my experiment,  I only get 20% validation accuracy during pre-training.

**Comment by Jeff-LiangF** [comment]
Hi, I just test the [pre-trained ckpt](https://drive.google.com/file/d/1YwcTJvASZJvn2LxyZZG4PcgXaBCKEv_4/view?usp=sharing) on ImageNet val set. What you should do is: 

1. replacing the [mae_vit model](https://github.com/enyac-group/supmae/blob/main/main_finetune.py#L225) to [models_supmae](https://github.com/enyac-group/supmae/blob/main/main_pretrain_supmae.py#L167).
2. Modify the [output of models_supmae](https://github.com/enyac-group/supmae/blob/main/models_supmae.py#L240) to ONLY return logits
3. You may want to change the [cls_head](https://github.com/enyac-group/supmae/blob/main/models_supmae.py#L68) all to projection_head because this is the format in pre-trained ckpt.
4. You may also want to change [no_weight_decay_list](https://github.com/enyac-group/supmae/blob/main/main_finetune.py#L282) to [] because supmae_model doesn't support `.no_weight_decay()` function

Using 0.75 mask ratio gives me: 
`* Acc@1 66.028 Acc@5 86.046`

Using 0.0 mask ratio gives me: 
`Acc@1 68.926 Acc@5 88.150`
